### PR TITLE
Cleanup of pipeline

### PR DESCRIPTION
**Pull request by rasmusjelsgaard**
Renaming of NuGet package name.
Improve the import of module in pester tests.
Make pipeline fail if tests fail.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,24 +15,22 @@ addons:
           key_url: "https://packages.microsoft.com/keys/microsoft.asc"
         packages:
             - powershell
-            - tree
 
 before_install:
-  - chmod ugo+x ./scripts/setup_repo.sh && ./scripts/setup_repo.sh
-  - tree /home/travis/build
-  - "echo \"work dir: $(pwd)\""
+  - chmod ugo+x ./scripts/setup_repo.sh
```
Comment (casz) [inline .travis.yml]: we could avoid this line by making it executable in git:
`git update-index --chmod=+x ./scripts/setup_repo.sh`

File: Tests/Invoke-Jenkins.Tests.ps1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
- Import-Module -Name $PSScriptRoot\..\ThreeShape.Jenkins.Beta.psm1
+ Import-Module -Name (Get-Item $PSScriptRoot\..\*.psm1)[0].FullName
```
Comment (casz) [inline Tests/Invoke-Jenkins.Tests.ps1]:
```suggestion
Import-Module -Name (Get-ChildItem $PSScriptRoot\..\*.psm1 | Select-Object -first 1).FullName
```

or

```suggestion
$module = (Get-ChildItem $PSScriptRoot\..\*.psm1 | Select-Object -first 1)
Import-Module -Name $module.FullName
```

File: scripts/setup_repo.sh [hunk 3]
```diff
@@ -13,6 +13,8 @@ echo "We will use this commit SHA: ${COMMIT_SHA}"
 echo "Travis build dir: ${TRAVIS_BUILD_DIR}"
 echo "Travis repo slog: ${TRAVIS_REPO_SLUG}"
```
Comment (casz) [inline scripts/setup_repo.sh]:
```suggestion
echo "Travis repo slug: ${TRAVIS_REPO_SLUG}"
```

File: Tests/Invoke-Jenkins.Tests.ps1 (hunk 2 above)
Comment (rasmusjelsgaard) [reply Tests/Invoke-Jenkins.Tests.ps1]: I don't believe option 2 will compile, actually. But the first suggestion is more readable and a definite improvement.

Comment (rasmusjelsgaard) [reply Tests/Invoke-Jenkins.Tests.ps1]: Done.

**Comment by zionyx** [comment]
Looks good to me at the current state.

**Pull request merged**